packages/core-mobile/app/ContextApp.tsx

```diff
@@ -48,7 +48,7 @@ const ContextApp = (): JSX.Element => {
 
   return (
     <Sentry.ErrorBoundary fallback={<TopLevelErrorFallback />}>
-      <StatusBar barStyle={'light-content'} backgroundColor="black" />
+      <StatusBar translucent backgroundColor="#00000000" />
```

Comment:
`ContextProviders` is created conditionally below, so it is not possible to get access to the theme at that point to get the `theme.transparent` variable here, but all it does is making the  `StatusBar` transparent so we can truly get edge to edge. Transparent is transparent, whatever the color.